dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+            });
+
+            it('Check called "this.isVisible" method with object argument', function () {
+                var row = {
+                    id: 123213
+                };
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
+                    return true;
+                });
+                overlayObj.isVisible(row);
+                expect(overlayObj.isVisible).toHaveBeenCalledWith(row);
+            });
+
+            it('Check isVisible and return false if id doesnt found', function () {
+                var row = {
+                    id: ''
+                };
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
+                    return false;
+                });
+                expect(overlayObj.isVisible(row)).toBe(false);
+            });
+
+            it('Check isVisible and return true if id found', function () {
+                var row = {
+                    id: 123213
+                };
+                overlayObj.isVisible = jasmine.createSpy().and.callFake(function () {
+                    return true;
+                });
+                expect(overlayObj.isVisible(row)).toBe(true);
+            });
+        });
+
+        describe('"getLabel" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('getLabel')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.getLabel;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+                var row ={
+                    id: 123213
+                };
+                expect(typeof overlayObj.getLabel(row)).toEqual('string');
+            });
+
+            it('Check called "this.getLabel" method with object argument', function () {
+                var row ={
+                    id: 123213
+                };
+                overlayObj.getLabel = jasmine.createSpy().and.callFake(function () {
+                    return '';
+                });
+                overlayObj.getLabel(row);
+                expect(overlayObj.getLabel).toHaveBeenCalledWith(row);
+            });
+
+            it('Check getLabel and return null if id doesnt found', function () {
+                var row ={
+                    id: ''
+                };
+                overlayObj.getLabel = jasmine.createSpy().and.callFake(function () {
+                    return '';
+                });
+                expect(overlayObj.getLabel(row)).toBe('');
+            });
+
+            it('Check getLabel and return string if id found', function () {
+                var row ={
+                    id: 123213
+                };
+                overlayObj.getLabel = jasmine.createSpy().and.callFake(function () {
```

Comment:
This test is not valuable as you are stubbing out the `getLabel` directly before testing it. Either don't stub the method you are testing, or remove this test.